en/flight_controller/pixhawk4_mini.md

```diff
@@ -0,0 +1,113 @@
+
+![Pixhawk 4 Mini interfaces](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_interfaces.png)
+
+> **Warning** The **RC IN** and **PPM** ports are for RC receivers only. These are powered! NEVER connect any servos, power supplies or batteries (or to any connected receiver).
+
+
+## Pinouts
+
+Download *Pixhawk 4 Mini* pinouts from [here](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/pixhawk4mini/pixhawk4mini_pinouts.pdf).
+
+## Dimensions
+
+![Pixhawk 4 Dimensions](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_dimensions.png)
+
+## Voltage Ratings
+
+*Pixhawk 4 Mini* can be redundant on the power supply if two power sources are supplied. The power rails are: **POWER** and **USB**.
+
+> **Note** The output power rail **MAIN OUT** does not power the flight controller board (and is not powered by it). You must supply power to one of **POWER** or **USB** or the board will be unpowered.
+
+**Normal Operation Maximum Ratings**
+
+Under these conditions all power sources will be used in this order to power the system:
+1. **POWER** (4.9V to 5.5V)
+1. **USB** input (4.75V to 5.25V)
+
+**Absolute Maximum Ratings**
+
+Under these conditions the system will not draw any power (will not be operational), but will remain intact.
+1. **POWER** input (operational range 4.1V to 5.7V, 0V to 10V undamaged)
+1. **USB** input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
+1. Servo input: VDD_SERVO pin of **MAIN OUT** (0V to 36V undamaged)
+
+
+## Assembly/Setup
+
+The [*Pixhawk 4 Mini* Wiring Quick Start](../assembly/quick_start_pixhawk4_mini.md) provides instructions on how to assemble required/important peripherals including GPS, Power Management Board, etc.
+
+
+## Building Firmware
+
+`make px4fmu-v5_default upload`
+
+
+## Debug Port
+
+The system's serial console and SWD interface runs on the **FMU Debug** port, while the I/O console and SWD interface can be accessed via **I/O Debug** port. In order to access these ports, the user must remove the *Pixhawk 4 Mini* casing.
+
+Both ports have standard serial pinout and can be connected to a standard FTDI cable (3.3V, but it's 5V tolerant) or a [Dronecode probe](https://kb.zubax.com/display/MAINKB/Dronecode+Probe+documentation). The pinout uses the standard Dronecode debug connector pinout. Please refer to the [wiring](https://dev.px4.io/en/debug/system_console.html) page for details of how to wire up this port.
+
+
+## Peripherals
+
+* [Digital Airspeed Sensor](https://drotek.com/shop/en/home/848-sdp3x-airspeed-sensor-kit-sdp33.html)
+* [Telemetry Radio Modules](https://docs.px4.io/en/telemetry/)
+* [Rangefinders/Distance sensors](https://docs.px4.io/en/sensor/rangefinders.html)
+
+
+## Supported Platforms / Airframes
+
+Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos. The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
```

Comment:
I am not sure this is true - see note above regarding the fact that this board does not have AUX ports.